site/app/views/LateDaysTableView.php

```diff
@@ -27,4 +27,18 @@ public function showLateTablePage(LateDays $late_days) {
             'grade_inquiry_enabled' => $this->core->getConfig()->isRegradeEnabled()
         ]);
     }
+
+    public function showLateTabletoInstructor(LateDays $late_days) {
+        $manage_url = $this->core->buildCourseUrl(['users']);
+        $this->core->getOutput()->addBreadcrumb('Manage Students', $manage_url);
+        $this->core->getOutput()->addBreadcrumb($late_days->getUser()->getDisplayFullName());
+        $this->core->getOutput()->addInternalCss('table.css');
+        $this->core->getOutput()->addInternalCss('latedaystableplugin.css');
+        $this->core->getOutput()->enableMobileViewport();
+        $preferred_name = $late_days->getUser()->getDisplayFullName();
```

Comment:
Is this used anywhere?

Reply:
Thanks for pointing out. Removed -> 2 less queries.